JULIERAJ/KIDS-FIRST-v2
# Bugfix/115/fix style login page

## One Line Description
fixed login page

## Requirements
-Style login page according to Figma design


## Test Steps
-Check if its align with figma design
- go to sign up page
- -enter email
- enter password
- click on show password and test if it works correctly
- click on login 

## UI/UX Outcome
- 
![image](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/107743756/aecfc8bb-6619-45cb-be3f-9812f76bc7b4)

![image](https://github.com/JULIERAJ/KIDS-FIRST-v2/assets/107743756/de4f803f-c944-430d-9b2d-38435aca51ad)

## Checklist
- Pull Request title includes the issue number and a brief description of the change.
- All changes should be tested and verified to work correctly.
- Code follows frontend best practices.
- Branch names follow the conventions in the contributing file.
- Commit messages follow the naming conventions in the contributing file.
